Program.cs

```diff
@@ -38,6 +39,7 @@ class Program
             new Product("Visual Studio 2022 Professional"     , "1299B4B9-DFCC-476D-98F0-F65A2B46C96D", "09662"),
         };
 
+        [STAThread]
```

Comment:
This should not be needed as you no longer use any COM components (for instance from Windows Forms)